### DISCUSSION TITLE
Welcome to Bitsteward Discussions!

### DISCUSSION THREAD

**Discussion by mxmvncnt** [Bitsteward/bitsteward#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mohamad-supangat** [comment]
is like the rofi-bitwarden client but using gtk?

**Reply by mxmvncnt** [reply]
I never used rofi-bitwarden so I can't answer you accurately. But bitsteward is an unofficial desktop GUI for bitwarden. It currently does not do much besides displaying your fault folders and items, but more should be coming as time passes!

**Reply by mohamad-supangat** [reply]
great project, the official gui is electron, and not run native on wayland
